auto_nag/scripts/pdfjs_update.py

```diff
@@ -38,6 +39,10 @@ def get_bz_params(self, date):
             "f2": "creation_ts",
             "o2": "greaterthan",
             "v2": start_date,
+            "n3": 1,
+            "f3": "blocked",
+            "o3": "changedby",
+            "v3": utils.get_config("common", "bot_bz_mail")[0],
```

Comment:
What if there is another valid bug? With this, the tool will always work only once.

Reply:
The bug IDs are always only mentioned in the first comment of the PDF.js update bugs